src/sentry/search/events/datasets/metrics.py

```diff
@@ -291,6 +313,36 @@ def function_converter(self) -> Mapping[str, fields.MetricsFunction]:
                     ),
                     default_result_type="integer",
                 ),
+                fields.MetricsFunction(
```

Comment:
They produce different queries, `uniq` just does `uniq(value)`, where `count_unique` uses a `uniqIf` on the metric id. I'm not sure if this makes a huge difference, but I'm trying to keep the queries between the two systems identical.

Reply:
What's the user query? `uniq(value)` without the conditional on metric_id is doing extra work since you're getting the unique count of a tag across multiple metrics

Reply:
^ Yep I second that

Reply:
snql looks like 
```
MATCH (metrics_sets) 
SELECT uniq(value) AS `count`, 
uniqIf(value, equals(tags[9223372036854776017], 9223372036854776009)) AS `crashed` 
WHERE project_id IN tuple(6296) 
AND org_id = 6030 
AND metric_id = 9223372036854775811 
GRANULARITY 10
```

Added a ticket to improve this later here: https://getsentry.atlassian.net/browse/WOR-1980